Name: Load Youtube Italian Feed
Version: 2.1.1-1
Author: iMarck90
Description: Load the italian feed for youtube app. Tested on Veer with webOS 2.1.1 maybe it's working also for webOS 2.2.x

--- .orig/usr/palm/applications/com.palm.app.youtube/app/models/yt-api.js
+++ /usr/palm/applications/com.palm.app.youtube/app/models/yt-api.js
@@ -200,7 +200,7 @@
 			return;
 		}
 					
-		var url = 'http://gdata.youtube.com/feeds/api/standardfeeds/most_popular?v=2&max-results=#{maxResults}&start-index=#{startindex}&time=this_week&alt=json&format=#{format}&safeSearch=strict&client=#{clientId}&orderby=published'.
+		var url = 'http://gdata.youtube.com/feeds/api/standardfeeds/IT/most_popular?v=2&max-results=#{maxResults}&start-index=#{startindex}&time=this_week&alt=json&format=#{format}&safeSearch=strict&client=#{clientId}&orderby=published'.
 			interpolate({
 			maxResults: numResults,
 			startindex: startIndex,
@@ -222,7 +222,7 @@
 			return;
 		}
 		
-		var url = 'http://gdata.youtube.com/feeds/api/standardfeeds/most_viewed?v=2&max-results=#{maxResults}&start-index=#{startindex}&time=this_week&alt=json&format=#{format}&safeSearch=strict&client=#{clientId}&orderby=viewCount'.
+		var url = 'http://gdata.youtube.com/feeds/api/standardfeeds/IT/most_viewed?v=2&max-results=#{maxResults}&start-index=#{startindex}&time=this_week&alt=json&format=#{format}&safeSearch=strict&client=#{clientId}&orderby=viewCount'.
 			interpolate({
 				maxResults: numResults,
 				startindex: startIndex,
